main.go

```diff
@@ -901,7 +899,7 @@ func convertWindupPerformToAnalyzer(perform windup.Iteration, where map[string]s
 	if perform.Technologyidentified != nil {
 		for _, ti := range perform.Technologyidentified {
 			for _, tag := range ti.Tag {
-				tags = append(tags, ti.Name+" - "+tag.Name)
+				tags = append(tags, ti.Name+"="+tag.Name)
```

Comment:
I don't think this matches the way windup formats the tags

Reply:
Hmm aren't we converting the category of tag into a flat string here?

Reply:
as I understand it, Windup has a nested structure for technology-identified element and we convert it into a flat string for us

Reply:
interesting, I looked at https://github.com/fabianvf/windup-rulesets-yaml/blob/41df7c4541e9f3ef902e2a9a8d6eb4c311437e1b/main.go#L452-L453 and thought we are just looking at suffix and prefix

Reply:
how exactly is the `-` evaluated?

Reply:
good point, I'd forgotten I did it that way, so the `=` vs `-` should be equivalent. What you'll get from that is tags like:

```
EAR Deployment=Other
EAR Deployment=Connect
EAR Deployment=Java EE
EJB XML=Connect
EJB XML=Bean
EJB XML=Java EE
```

(these are from a single run of one of the rulesets). If that's what's intended then lgtm

Reply:
@fabianvf I actually want them the other way, `EAR Deployment` in this case is actually a tag whereas `Connect`, `Other` and `Java EE` are categories (TagTypes) it belongs to.